.github/workflows/ci.yml

```diff
@@ -126,12 +126,12 @@ jobs:
     - run: ./gradlew clean assemble -Dbuild.version_qualifier=${{ env.TEST_QUALIFIER }} && test -s ./build/distributions/opensearch-security-${{ env.SECURITY_PLUGIN_VERSION_ONLY_NUMBER }}-${{ env.TEST_QUALIFIER }}-SNAPSHOT.zip
 
     - run:  |
-        ## EXISTING_OS_VERSION outputs the major version, example as 2
-        EXISTING_OS_VERSION=$(./gradlew properties | grep opensearch.version | cut -d':' -f2- | awk '{$1=$1};1' | cut -d '-' -f1 | cut -d '.' -f1)
-        ## INCREMENT_OS_VERSION in an increment of 1, example if EXISTING_OS_VERSION is 2, INCREMENT_OS_VERSION is 3
-        INCREMENT_OS_VERSION=$((++EXISTING_OS_VERSION)) 
-        ./gradlew clean updateVersion -DnewVersion=$INCREMENT_OS_VERSION.0.0-SNAPSHOT
-        test `./gradlew properties | grep opensearch.version | cut -d':' -f2- | awk '{$1=$1};1'` = $INCREMENT_OS_VERSION.0.0-SNAPSHOT
+        ## Make sure the current doesn't match the test version
+        test "$(./gradlew properties | grep opensearch.version)" != "opensearch_version: 2.1.0-SNAPSHOT"
```

Comment:
@peternied I am wondering what is special about `2.1.0-SNAPSHOT` version? shouldn't it be aligned with `opensearch.version`?

Reply:
Is this to test the version increment automation at build time? If so, should we create a meaningless tag like `VERIFY-VERSION-INCREMENT`?

Reply:
Will this change be reverted to original at some point?

Reply:
This value should never be changed

Reply:
I've moved this to a variable to make this more apparent in the workflow description.